src/renderer/components/NewProjectModal/ImportMediaView.tsx

```diff
@@ -4,19 +4,18 @@ import { Box, Button, Stack, styled, Typography } from '@mui/material';
 import colors from 'renderer/colors';
 import CloseIcon from '@mui/icons-material/Close';
 import IconButton from '@mui/material/IconButton';
-import { ApplicationStore } from 'renderer/store/sharedHelpers';
-import { projectCreated } from 'renderer/store/currentProject/actions';
-import { recentProjectAdded } from 'renderer/store/recentProjects/actions';
-import { Project } from 'sharedTypes';
+import { ApplicationStore } from '../../store/sharedHelpers';
+import { projectCreated } from '../../store/currentProject/actions';
+import { Project } from '../../../sharedTypes';
```

Comment:
Appreciate the cleanups, we should also put somewhere that this is the preferred styling. Maybe in #process or something.

Reply:
Yeah we could automate this, idk if people prefer absolute or relative though